docs/Shaders/Perspective.md

```diff
@@ -0,0 +1,59 @@
+# Perspective
+
+With the Perspective shader you can add a lighting & 3D effect to the to be rendered texture.
+
+## Usage
+
+If you want to use shaders that come with the Lightning framework you need to import Lightning first.
+
+```js
+import Lightning from '@lightningjs/core'
+```
+
+### Initialize
+
+To use the Perspective shader you need to apply it to the shader property of an element:
+
+```js
+//in template
+class MyApp extends Lightning.Application {
+    static _template() {
+        return {
+            Picture: {w: 300, h: 300, src: 'MyImage.jpg', shader: {type: Lightning.shaders.Perspective, rx: Math.PI * 0.25}}
```

Comment:
Maybe we should explain a bit better what happens here with `Math.PI * 0.25`?

Reply:
added extra explanation how rotation works